src/ray/raylet/scheduling_queue.cc

```diff
@@ -8,8 +8,8 @@ namespace {
 
 // Helper function to remove tasks in the given set of task_ids from a
 // queue, and append them to the given vector removed_tasks.
-void RemoveTasksFromQueue(ray::raylet::TaskState task_state,
-                          ray::raylet::SchedulingQueue::TaskQueue &queue,
+template <typename TaskQueue>
+void RemoveTasksFromQueue(ray::raylet::TaskState task_state, TaskQueue &queue,
                           std::unordered_set<ray::TaskID> &task_ids,
                           std::vector<ray::raylet::Task> &removed_tasks,
```

Comment:
here, to be consistent, instead of `std::vector<ray::raylet::Task> &removed_tasks` , should we switch to `std::vector<ray::raylet::Task> *removed_tasks` (since you are doing it in `TaskQueue::RemoveTask`)?

Reply:
Good idea, I'm making the change.